install_ddev.sh

```diff
@@ -14,26 +14,39 @@ BINOWNER=$(ls -ld /usr/local/bin | awk '{print $3}')
 USER=$(whoami)
 SHACMD=""
 FILEBASE=""
+CURRENT_DIR=$PWD
 
 if [[ "$OS" == "Darwin" ]]; then
     FILEBASE="ddev_macos"
+
+    if ! docker --version >/dev/null 2>&1; then
+        # Install and open Docker
+        hdiutil attach -nobrowse "${CURRENT_DIR}/docker_installs/Docker.dmg"
+        sleep 10
+        cp -rp /Volumes/Docker/Docker.app /Applications/
+        wait
+        open -a /Applications/Docker.app
+        hdiutil detach /Volumes/Docker
+    fi
+
 elif [[ "$OS" == "Linux" ]]; then
     FILEBASE="ddev_linux"
+
+    if ! docker --version >/dev/null 2>&1; then
+        printf "${YELLOW}Docker is required for ddev. Download and install docker at https://www.docker.com/community-edition#/download before attempting to use ddev.${RESET}\n"
+    fi
+
+    if ! docker-compose --version >/dev/null 2>&1; then
+        printf "${YELLOW}Docker Compose is required for ddev. Download and install docker-compose at https://www.docker.com/community-edition#/download before attempting to use ddev.${RESET}\n"
+    fi
+    
 else
     printf "${RED}Sorry, this installer does not support your platform at this time.${RESET}\n"
     exit 1
 fi
 
-if ! docker --version >/dev/null 2>&1; then
-    printf "${YELLOW}Docker is required for ddev. Download and install docker at https://www.docker.com/community-edition#/download before attempting to use ddev.${RESET}\n"
-fi
-
-if ! docker-compose --version >/dev/null 2>&1; then
-    printf "${YELLOW}Docker Compose is required for ddev. Download and install docker-compose at https://www.docker.com/community-edition#/download before attempting to use ddev.${RESET}\n"
-fi
-
 echo "Installing docker images for ddev to use..."
-gzip -dc $(ls ddev_tarballs/ddev_docker_images*.tar.gz) | docker load
+tar -xOpvf $(ls ddev_tarballs/ddev_docker_images*.tar.xz) | docker load
```

Comment:
This isn't right, as the input to docker load is a stream of tar, so we don't want to just untar, we have to *uncompress* and pass that to `docker load`